src/features/unified-share-modal/UnifiedShareModal.js

```diff
@@ -90,6 +90,8 @@ type Props = {
      * This function should return a Promise.
      */
     sendSharedLinkError: React.Node,
+    /** Function hoists contact data upon updates to the parent component. Only needed for suggested collabs. */
+    setUpdatedContacts?: Function,
```

Comment:
Can this be made more specific?

Reply:
Sure I'll update this and get someone who has worked on this feature before to take a look.